torch/_meta_registrations.py

```diff
@@ -211,6 +211,23 @@ def meta_bernoulli(self, *, generator=None, out):
     return out
 
 
+@register_meta(aten._to_copy.default, False)
+def _to_copy(
+    self: torch.Tensor,
+    dtype=None,
+    layout=None,
+    device=None,
+    pin_memory=None,
+    memory_format=None,
+):
+    dtype = self.dtype if dtype is None else dtype
+    device = self.device if device is None else device
+    layout = self.layout if layout is None else layout
+    assert pin_memory is None
+    assert memory_format is None
+    return self.new_empty(self.shape, dtype=dtype, device=device, pin_memory=pin_memory)
```

Comment:
just curious - is this python decomp needed now? (aka something now breaks with the C++ decomp)

Reply:
`_to_copy` isn't `CompositeImplicitAutograd`, so I don't think there's a "C++ decomp"? Or do you mean the meta function implemented in C++?

Reply:
oh yep, I mean the meta function implemented in C++; is that not useable with the changes in this PR?

Reply:
I don't think we use any C++ meta functions today, although I think it's plausible we could in the future.

Reply:
This implementation is slightly incorrect and it seems there were no tests to catch that: https://github.com/pytorch/pytorch/issues/84335.